platform-includes/getting-started-use/javascript.nestjs.mdx

```diff
@@ -72,7 +72,10 @@ import { SentryGlobalFilter } from '@sentry/nestjs/setup';
 export class AppModule {}
 ```
 
-**Note:** In NestJS + GraphQL applications replace the `SentryGlobalFilter` with the `SentryGlobalGraphQLFilter`.
+{/* TODO(v9): Remove this note. */}
+
+**Note:** If you have a NestJS + GraphQL application and you are using the `@sentry/nestjs` SDK version `8.38.0` or earlier, replace the `SentryGlobalFilter` with the `SentryGlobalGraphQLFilter`.
```

Comment:
```suggestion
**Note:** If you have a NestJS + GraphQL application and you are using the `@sentry/nestjs` SDK version `8.38.0` or earlier, replace the `SentryGlobalFilter` with the `SentryGlobalGenericFilter`.
```

I don't think we should suggest using SentryGlobalGraphQLFilter at all (see https://github.com/getsentry/sentry-docs/pull/11830).

Reply:
Thanks for raising! Updated.